### ISSUE TITLE
Can we run other cpu-types in CFU Playground (initially w/o CFU)?

### ISSUE THREAD

**Issue by tcal-x** [google/CFU-Playground#699]
There are some (many?) places in CFU Playground that assume the CPU is VexRiscv.  What if any work would need to be done to allow other Litex-supported CPUs (SERV, PicoRV32, corev*, etc) to be dropped into the SoC and run the CFU Playground software on them?

This could be useful for benchmarking initially, and pave the way for work of adding compatible CFU interfaces to these other CPUs.

FYI @ShvetankPrakash

**Comment by ShvetankPrakash** [comment]
Yes, I think this is very interesting for other directions and would be cool to see if it works!
